app/views/races/_create.slim

```diff
@@ -1,7 +1,7 @@
 - game = local_assigns.fetch(:game, Game.new)
 - category = local_assigns.fetch(:category, Category.new)
 
-// Make sure to have #vue-race be defined before calling this partial
+// Make sure to have #vue defined before calling this partial
```

Comment:
Is this comment relevant now that `vue` is defined in `application.slim`?

Reply:
Good point, no it is not!